### PR TITLE
usernameAdd

### PR DESCRIPTION
**Pull request by LeoUCon**
## Linked Issue

#13 

## Description

Added my GitHub username to the end of [`src/_data/members/members.json`](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/src/_data/members/members.json)

## Methodology

Changes made to be part of Virtual Coffee.

### CODE REVIEW

File: src/_data/members/members.json [hunk 1]
```diff
@@ -71,5 +71,6 @@
   "pedaars",
   "awildstone",
   "danieljanderson",
-  "rek990"
+  "rek990",
+  "mleouc"
```
Comment (danieltott) [inline src/_data/members/members.json:75]: This should be your GitHub user name - looks like it's `itsLeoUC`

**Review by danieltott — CHANGES_REQUESTED** [review]
Thanks for the PR @itsLeoUC! Just one tweak and then we can get it merged!

File: src/_data/members/members.json (hunk 1 above)
Comment (LeoUCon) [reply src/_data/members/members.json]: Thanks for the review, I have made the adjustment with the right username. I was looking for a better username 😃, now I am good with the current one `LeoUCon`

**Comment by danieltott** [comment]
welcome

**Pull request merged**